notebooks/tm/landcover/01-extract-landcover-features.ipynb

```diff
@@ -0,0 +1,1084 @@
+{
```

Comment:
<p>Please add 2 cells here: (1) a cell that plots a sample landcover map for a baranggay (pick a big one) and (2) a cell that filters <code>result</code> and displays the corresponding row for that baranggay</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>